napalm/ios/ios.py

```diff
@@ -1088,7 +1088,8 @@ def get_interfaces(self):
 
             interface_regex_1 = r"^(\S+?)\s+is\s+(.+?),\s+line\s+protocol\s+is\s+(\S+)"
             interface_regex_2 = r"^(\S+)\s+is\s+(up|down)"
-            for pattern in (interface_regex_1, interface_regex_2):
+            interface_regex_3 = r"^(Control Plane Interface)\s+is\s+(.+?),\s+line\s+protocol\s+is\s+(\S+)"
```

Comment:
The lint check is failing because of this long line.
```suggestion
            interface_regex_3 = r"^(Control Plane Interface)\s+is\s+(.+?),\s+line\s+protocol\s+is\s+(\S+)"
            interface_regex_3 = (
                r"^(Control Plane Interface)"
                r"\s+is\s+(.+?),\s+line\s+protocol\s+is\s+(\S+)"
            )
```